ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -323,34 +319,14 @@ private[commands] class CommandTracker[Context](
         }
 
         val trackedCommandKeys = pendingCommandKeys(maybeSubmissionId, commandId)
```

Comment:
Minor: If we're running this `CommandTracker` with `SubmissionIdPropagationMode.Supported`, then we'll do a full scan over the map of tracked commands to compute the `trackedCommandKeys` even though we know the result of that computation will not be used. That's a bit inefficient.

Reply:
I just realized that it's broken, as we also remove the tracked commands here.

Reply:
Sorted out